modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/model/Detector.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.modelservice.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Detector {
+    private UUID id;
```

Comment:
This should be `uuid` given that it's that way everywhere else in the system. I think you mentioned that we have to update something on the ES side--not sure. Can we rename this field to `uuid` and do some kind of annotation-based mapping to map to `id` in the interim?

Reply:
After conversation with you it sounds like this will involve reimporting the data into the system. I am OK with that since I want to solve this before we have Docker Compose in place (which will make it easier for people to run this themselves and hence lock us in more). However you don't need to do it yourself--Karan and I can do it when we migrate the detectors over.

Reply:
Sure, this could be taken up as a separate change.